.github/workflows/cherrypick.yaml

```diff
@@ -5,6 +5,9 @@ on:
       - main
     types: ["closed"]
 
+permissions:
```

Comment:
with `read`, it's able to create PR from action?

Reply:
tbh, im not sure if this action was ever working, merging this one to rm the codeql alerts, and will test this out